### DISCUSSION TITLE
[breejs / ts-worker] typescript compiler configuration

### DISCUSSION THREAD

**Discussion by rhazegh** [breejs/bree#199] [Q&A] [answered]
I am looking into using [@breejs/ts-worker](https://github.com/breejs/ts-worker) but couldn't find much info about it in the repo. 

- Does `ts-worker` use the already existing `tsconfig.json` in the root folder? If not, how do you configure the typescript compiler behavior?
- Is it possible to use a specific `tsconfig.special.json` file  to configure the behavior of the typescript compiler?

Thanks.

**Comment by shadowgate15** [comment, marked as answer]
`@breejs/ts-worker` uses [ts-node](https://www.npmjs.com/package/ts-node) `register` api to load the worker. however it doesn't take any options. Right now. this might be something we should add.

**Reply by shadowgate15** [reply]
It will use the nearest `tsconfig.json` by default.
